app/Http/Controllers/PostController.php

```diff
@@ -263,6 +263,39 @@ public function postBestAnswer(Request $request)
         }
     }
 
+    public function postReport(Request $request)
+    {
+        $input = $request->all();
+
+        if ($request->ajax() && !empty($input['post_id']) && Auth::check()) {
+            $input['user_id'] = Auth::id();
+            $input['comment'] = config('constants.REPORT_MESSAGES.' . $input['report'], null);
+            $input['comment'] = $input['comment'] . '|||' .$input['note'];
+
+            $validator = \Validator::make($input,
+            [
+                'note' => 'max:200',
+                'report' => 'required|numeric',
+            ],
+            [
+                'max' => __('validation.max.string', ['attribute' => __('page.post.note'), 'max' => 200]),
+                'report.required' => __('validation.required', ['attribute' => __('page.post.report')]),
+            ]);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/PostController.php</br>**Line**: 283</br>Closing parenthesis of a multi-line function call must be on a line by itself